lib/logstash/inputs/tcp.rb

```diff
@@ -112,6 +112,13 @@ class LogStash::Inputs::Tcp < LogStash::Inputs::Base
   # All the certificates will be read and added to the trust store.
   config :ssl_certificate_authorities, :validate => :array, :default => []
 
+  # NOTE: the default setting [] uses Java SSL engine defaults.
+  config :ssl_supported_protocols, :validate => ['TLSv1.1', 'TLSv1.2', 'TLSv1.3'], :default => [], :list => true
```

Comment:
ES (still) supports legacy protocols as well I believe it would make sense to support `SSLv3` and `TLSv1` as well

Reply:
```suggestion
  config :ssl_supported_protocols, :validate => ['SSLv3', 'TLSv1', 'TLSv1.1', 'TLSv1.2', 'TLSv1.3'], :default => [], :list => true
```